src/mscorlib/Resources/Strings.resx

```diff
@@ -413,7 +413,7 @@
     <value>Search pattern '{0}' cannot contain ".." to move up directories and can be contained only internally in file/directory names, as in "a..b".</value>
   </data>
   <data name="Arg_InvalidTypeInRetType" xml:space="preserve">
-    <value>The return Type contains some invalid type (i.e. null, ByRef)</value>
+    <value>The return Type contains some invalid type (i.e. null)</value>
```

Comment:
This error message is a bit misleading - if you pass in `null` return type type, everything will work fine actually. It will be treated as null.

It may be better to change it to `The return Type must be a type provided by the runtime.`.

Reply:
done